glow_test.go

```diff
@@ -16,6 +17,13 @@ func TestGlowSources(t *testing.T) {
 
 	for _, v := range tt {
 		t.Run(v, func(t *testing.T) {
+			// Start by checking for network issues.
+			_, nerr := readmeURL(v)
+			if nerr != nil && strings.Contains(nerr.Error(), "no such host") {
+				t.Logf("Error during execution (args: %s):\n%v", v, nerr)
+				t.Skip("Test uses network. Are you connected to the Internet?")
+			}
```

Comment:
Please try this, that would be safer

```suggestion
			// Start by checking for network issues.
			_, err := readmeURL(v)
			var dnsError *net.DNSError
			if errors.As(err, &dnsError) {
				t.Logf("Error during execution (args: %s):\n%v", v, err)
				t.Skip("Test uses network. Are you connected to the Internet?")
			}
```

This would be a more idiomatic way.

Reply:
I didn't test it, I simply wrote it after checking http.Get stack

I'm unsure if it has to be `if errors.As(err, &dnsError) {` or `if errors.As(err, dnsError) {` as dnsError is a pointer.

I let you check and adapt

Reply:
Thank you!! This is very helpful :)